homeassistant/components/lidarr/__init__.py

```diff
@@ -80,6 +80,6 @@ def __init__(
             entry_type=DeviceEntryType.SERVICE,
             identifiers={(DOMAIN, coordinator.config_entry.entry_id)},
             manufacturer=DEFAULT_NAME,
```

Comment:
Right now the name is only used as the manufacture name, I think it would be better to either drop this const or rename it to `MANUFACTURER_NAME`

Reply:
What about the `config_flow`? `DEFAULT_NAME` is also used there.